alsid-syslog-proxy/azuredeploy.json

```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "minLength": 1,
+            "type": "String"
+        },
+        "vmAdminUserName": {
+            "minLength": 1,
+            "type": "String"
+        },
+        "vmAdminPassword": {
+            "type": "SecureString"
+        },
+        "dnsLabelPrefix": {
```

Comment:
add a defaultValue using uniqueString()

Reply:
uniqueString can start with a number so need to account for that (prepend a char)